README.md

```diff
@@ -49,34 +49,42 @@
 In order to have the icons displayed correctly please use/update your favorite [patched font](https://www.nerdfonts.com/font-downloads).
 If you do not have a patched font installed, you can override or remove any icon. Check the documentation below on the options available.
 
+### Manual (Recommended)
+
+This method is recommended as TPM has some issues with name conflicts.
+
+1. Clone this repository to your desired location (e.g.
+   `~/.config/tmux/plugins/catppuccin`).
+
+```bash
+mkdir -p ~/.config/tmux/plugins/catppuccin
+git clone https://github.com/catppuccin/tmux.git ~/.config/tmux/plugins/catppuccin/tmux
+```
+
+2. Add the following line to your `tmux.conf` file:
+    `run ~/.config/tmux/plugins/catppuccin/tmux/catppuccin.tmux`
```

Comment:
The options _cannot_ be sourced inside the main config. This is because the main config and everything it sources uses `%if` statements, which are resolved at evaluation time. Since those `%if` statements require the variables to already be set, they will not resolve correctly, hence the options need to be calculated completely _before_ the main config is evaluated.

Reply:
Maybe it's worth adding a `catppuccin.conf`
![image](https://github.com/user-attachments/assets/3c54efb4-9366-475e-bf28-9afddeb7935a)

Reply:
Honestly, I don't think that the added complexity justifies a 20ms bump from 40ms. We could load the config as is and maintain a frame rate of 25fps, which is fast enough for people not to notice. Also it's a terminal, and this command is run once at tmux startup. I'm not going to spend time worrying about this.

Reply:
No problem will probably add it myself at some time and update the manual method in the readme since it is in my eyes complexity free.